src/webgpu/shader/execution/expression/call/builtin/abs.spec.ts

```diff
@@ -143,16 +163,13 @@ If e evaluates to the largest negative value, then the result is e.
   });
 
 g.test('f32')
-  .specURL('https://www.w3.org/TR/2021/WD-WGSL-20210929/#float-builtin-functions')
+  .specURL('https://www.w3.org/TR/WGSL/#float-builtin-functions')
   .desc(
     `
-float abs:
```

Comment:
Should there be unimplemented tests for AbstractFloat and f16 like there is for AbstractInt?